DOCS/history-OSI-SPDX.md

```diff
@@ -0,0 +1,87 @@
+# A History of OSI-approved licenses and the SPDX License List
+
+In the first releases of the SPDX License List, it was determined that the SPDX License List should include all licenses that were ever OSI-approved. (A policy that SPDX-legal maintains to this day.) The rationale being that once OSI-approved, always OSI-approved and that even old or "deprecated" (by way of a newer version or other such circumstance) licenses may still appear "in the wild". (Note: the OSI and SPDX use the term "deprecated" in different ways.)
+
+In 2011, OSI [began using the SPDX license identifiers](https://opensource.org/minutes20110608) in their URLs and on the license pages.
```

Comment:
Link appears to be incorrect now; should be https://opensource.org/meeting-minutes/minutes20110608